src/material/list/selection-list.spec.ts

```diff
@@ -484,7 +484,7 @@ describe('MatSelectionList without forms', () => {
       expect(manager.activeItemIndex).toBe(-1);
 
       const event = createKeyboardEvent('keydown', END);
-      Object.defineProperty(event, 'shiftKey', { get: () => true });
+      Object.defineProperty(event, 'altKey', { get: () => true });
```

Comment:
Just for context: We had to change this test because previously we didn't allow HOME/END with any modifier keys pressed. Now though we will start supporting those keys with the `SHIFT` modifier key, right?

Reply:
Yes, exactly. I did it since, from what I can tell, home and end don't do much with a modifier anyway.